src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -117,8 +117,33 @@
 import net.sourceforge.kolmafia.request.CampgroundRequest.CropType;
 import net.sourceforge.kolmafia.request.DeckOfEveryCardRequest.EveryCard;
 import net.sourceforge.kolmafia.request.FloristRequest.Florist;
-import net.sourceforge.kolmafia.session.*;
+import net.sourceforge.kolmafia.session.BanishManager;
```

Comment:
not a concern for this PR, but some people's editors appear to be collapsing many imports into *, while this here is expanding * into many. Can we agree on a standard setting (e.g. if greater than X, star, otherwise not, tests can always use * if that would be more convenient?)

Reply:
My impression is that we don't want stars - although, why not for tests?
For actual code, make sure that you know you are adding a new dependency.
But let's discuss this elsewhere.

Reply:
My argument: if you see a method call for another package, you can see exactly where to find it in the imports - unless it is hiding behind a star.

I use Emacs, not an IDE, so this is rather important for me.